irida_staramr_results/cli.py

```diff
@@ -38,27 +110,16 @@ def main():
         sys.exit(1)
```

Comment:
Is this sys.argv check still needed? We have required arguments so it should validate that for us if there are no args given.

Reply:
Done. It's not needed, removed the check.